client/src/views/sidebar-menu/SidebarMenuPage.vue

```diff
@@ -20,48 +20,67 @@
-                    <ms-image
-                      v-else
-                      :image="LogoIconGradient"
-                      class="organization-avatar-logo"
-                    />
-                  </ion-avatar>
-                  <div class="organization-text">
-                    <ion-card-title class="title-h3">
-                      {{ userInfo?.organizationId }}
-                    </ion-card-title>
-                  </div>
+                <ion-avatar class="organization-avatar body-lg">
+                  <span v-if="!isTrialOrg">{{ userInfo ? userInfo.organizationId.substring(0, 2) : '' }}</span>
+                  <ms-image
+                    v-else
+                    :image="LogoIconGradient"
+                    class="organization-avatar-logo"
+                  />
+                </ion-avatar>
+                <div class="organization-text">
+                  <ion-card-title class="title-h3">
+                    {{ userInfo?.organizationId }}
+                  </ion-card-title>
                 </div>
-                <div
+                <ms-image
+                  :image="ChevronExpand"
                   class="header-icon"
-                  v-show="isDesktop()"
-                >
-                  <ms-image :image="ChevronExpand" />
-                </div>
+                />
               </ion-card-header>
 
               <div
-                class="organization-card-manageBtn"
+                class="organization-card-buttons"
                 v-show="userInfo && userInfo.currentProfile != UserProfile.Outsider"
-                @click="navigateTo(Routes.Users)"
               >
                 <ion-text
-                  class="button-medium"
+                  @click="navigateTo(Routes.Users)"
+                  class="organization-card-buttons__item button-medium"
+                  id="manageOrganization"
                   button
                 >
-                  {{
-                    userInfo && userInfo.currentProfile === UserProfile.Admin
-                      ? $msTranslate('SideMenu.manageOrganization')
-                      : $msTranslate('SideMenu.organizationInfo')
-                  }}
+                  <ion-icon
+                    class="button-icon"
+                    :icon="userInfo && userInfo.currentProfile === UserProfile.Admin ? cog : informationCircle"
+                  />
+                  <span>
+                    {{
+                      userInfo && userInfo.currentProfile === UserProfile.Admin
+                        ? $msTranslate('SideMenu.manageOrganization')
+                        : $msTranslate('SideMenu.organizationInfo')
+                    }}
+                  </span>
+                </ion-text>
+                <ion-text
+                  class="organization-card-buttons__item button-medium"
+                  id="goHome"
+                  button
+                  @click="goToHome"
+                  v-if="!isWeb()"
```

Comment:
```suggestion
                  v-if="isDesktop()"
```
`!isWeb()` implicitly includes any other platform. Is doesn't matter for now but it's more rigorous.